questionnaire/questionnaire-service.js

```diff
@@ -167,7 +170,10 @@ function createQuestionnaireService({
                 }
 
                 if (task.type === 'sendSms') {
-                    if (pointer.has(questionnaire, task.templatePlaceholderMap.phoneNumber)) {
+                    if (
+                        pointer.get(questionnaire, task.templatePlaceholderMap.phoneNumber) !==
+                        undefined
```

Comment:
not sure what's going on with the formatting?

Reply:
The max line width is 80 columns. So it'll wrap this

Reply:
Cheers, I thought it was the line wrap style I am just not used to how it looks, 
pointer.get(questionnaire, task.templatePlaceholderMap.phoneNumber)
       !== undefined
[Checkstyle](https://checkstyle.sourceforge.io/apidocs/index.html?com/puppycrawl/tools/checkstyle/checks/whitespace/OperatorWrapCheck.html)

Reply:
as opposed to [JavaScript Style Guide](https://www.w3schools.com/js/js_conventions.asp)